src/fetchtastic/client_app_config.py

```
@@ -0,0 +1,200 @@
+
+
+def _coerce_keep_count(value: Any, default: int = DEFAULT_APP_VERSIONS_TO_KEEP) -> int:
+    """Coerce a keep-count config value to a non-negative integer."""
+    try:
+        return max(0, int(value))
+    except (TypeError, ValueError):
+        return int(default)
+
+
+def _classify_selected_assets(
+    config: dict[str, Any],
```

Ruff (0.15.12)

[warning] Unused function argument: `config`

(ARG001)